homeassistant/components/sensor/ted5000.py

```diff
@@ -22,13 +22,14 @@
 
 DEFAULT_NAME = 'ted'
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=3)
 
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Optional(CONF_PORT, default=80): cv.port,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional('show_unavailable', default=True): cv.boolean,
```

Comment:
This should not be an option. It's going to have the property unavailable as `true` when it's unavailable, no config option/

Reply:
What if I want to get a 0W or 0V value whenever there's a MTU disconnected?

Reply:
The MTU does exists and is configured on TED5000 interface. For example, my pool breaker trips, I don't want it as unavailable, I want it 0V and 0W...

Reply:
And above all, it's not a required field and it's active by default: It's going to have to return the property available as `false` when it's unavailable.